inject.js

```diff
@@ -1,19 +1,21 @@
 chrome.extension.sendMessage({}, function(response) {
   var tc = {
     settings: {
-      speed: 1.0,           // default 1x
-      resetSpeed: 1.0,      // default 1x
-      speedStep: 0.1,       // default 0.1x
-      rewindTime: 10,       // default 10s
-      advanceTime: 10,      // default 10s
-      resetKeyCode:  82,    // default: R
-      slowerKeyCode: 83,    // default: S
-      fasterKeyCode: 68,    // default: D
-      rewindKeyCode: 90,    // default: Z
-      advanceKeyCode: 88,   // default: X
-      displayKeyCode: 86,   // default: V
-      rememberSpeed: false, // default: false
-      startHidden: false,   // default: false
+      speed: 1.0,                    // default 1x
+      resetSpeed: 1.0,               // default 1x
+      speedStep: 0.1,                // default 0.1x
+      fastSpeed: 2.5,                // default 2.5x
```

Comment:
As a default, I think most folks will find this too fast. I recommend we set 1.75 as the default, which is the recommended "comfortable" speed for most folks based on past research.